app/mailers/wishlist_mailer.rb

```diff
@@ -1,12 +1,13 @@
 class WishlistMailer < ApplicationMailer
-  def wish_list_creation_email(user)
-    @user = user
+  #he purpose of the wishlist creation email is to give confirmation to the therapist that the wishlist was successfully created
+  def wish_list_creation_email(donor)
+    @donor = donor
     @url  = login_url
-    mail(to: @user.email, subject: 'Congratulations for creating a wishlist!')
+    mail(to: @donor.email, subject: 'Congratulations for creating a wishlist!')
   end
-  def item_purchased_email(user)
-    @user = user
+  def item_purchased_email(donor)
+    @donor = donor
     @url  = login_url
-    mail(to: @user.email, subject: 'You have purchased an item you reserved')
+    mail(to: @donor.email, subject: 'You have purchased an item you reserved')
```

Comment:
email body is fine, just suggesting a change for the subject

we may want to remove "for a snowman" and refer to "its recipient" instead of "your secret snowman" though, since wediko doesn't want to be stuck with only doing secret snowman events